docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.5'
+
+services:
+  contracts-env:
+    security_opt:
+      - no-new-privileges
+    user: node
```

Comment:
If you are mounting a host directory, I think you need the previous user mapping you had, otherwise files built in the container will be mapped to root on the host.